cmd/seednode/config/p2pKey.pem

```diff
@@ -0,0 +1,4 @@
+-----BEGIN PRIVATE KEY for 16Uiu2HAmKBCtAqnmawCJ5nxd9AccU3fj4sjA8WA3oksqeZgcsJJv-----
```

Comment:
can you generate the seed node pem file from the testnet scripts, into the relevant seednode folder?